packs/apis/trace/config.yml

```diff
@@ -0,0 +1,18 @@
+id: 07f0b96c-add9-43d5-962d-1865d0149082
+name: Trace and Span API
+description: >-
+  Our Trace API is used to send distributed tracing data to New Relic: either in our own generic format or the Zipkin data format. This API is also how trace data from some of our integrations and exporters is reported to New Relic.
+summary: >-
+  Our Trace API is used to send distributed tracing data to New Relic: either in our own generic format or the Zipkin data format. This API is also how trace data from some of our integrations and exporters is reported to New Relic.
+icon: icon.png
+logo: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Trace and Span API
+documentation:
+  - name: Trace and Span API usage docs
+    description: >-
+      Our Trace API is used to send distributed tracing data to New Relic: either in our own generic format or the Zipkin data format. This API is also how trace data from some of our integrations and exporters is reported to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-athena-monitoring-integration
```

Comment:
Wrong copy/paste? This link is referring to AWS Athena

Reply:
Forgot to save 😵‍💫 Good catch, thanks @polfliet